types/n3/tsconfig.json

```diff
@@ -5,6 +5,7 @@
             "es6",
             "dom"
         ],
+        "target": "ES2015",
```

Comment:
What is the reason for this?

Reply:
@mielvds ?

Reply:
I wasn't able to run the tests without

Reply:
Which change required it?

The test suite should be passing with the original config before any of the changes in this PR were made - unless there have been changes to the DT setup that I don't know about.

Reply:
Right... I got `ERROR: 270:21  expect  TypeScript@5.1 compile error: 
Type 'Generator<Quad, any, unknown>' can only be iterated through when using the '--downlevelIteration' flag or with a '--target' of 'es2015' or higher.` without

Reply:
@jeswr @RubenVerborgh not sure how to solve this one. Suggestions?

Reply:
Per https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65275#issuecomment-1614124038 I would just make the return type `Iterable<OutQuad>`

Reply:
done in 50ac2831d2ba52f3fc2c6401bdefb0ecca3e03cb